src/NuGet.Core/NuGet.Build.Tasks.Pack/NuGet.Build.Tasks.Pack.targets

```diff
@@ -407,7 +409,7 @@ Copyright (c) .NET Foundation. All rights reserved.
       </TfmSpecificFrameworkAssemblyReferences>
     </ItemGroup>
   </Target>
-  
+
```

Comment:
Remove the space changes, since it adds no value to the changeset.

Reply:
Will do clean up at the end.

Reply:
Sounds good, let us know when you are done so we can review again.